server/graphql/v2/interface/Account.ts

```diff
@@ -862,6 +871,14 @@ export const AccountFields = {
   },
   transactions: accountTransactions,
   orders: accountOrders,
+  expenses: {
+    type: new GraphQLNonNull(GraphQLExpenseCollection),
+    args: ExpensesCollectionQueryArgs,
+    resolve(collective, args, req) {
+      args.fromAccount = { legacyId: collective.id };
+      return ExpensesCollectionQueryResolver(undefined, args, req);
+    },
+  },
```

Comment:
It's unclear to me whether a field called `account.expenses` should return received expenses, submitted expenses, or both. IMO, we should either rename the field to make it explicit or add a param (e.g. `direction`) to clarify.